### ISSUE TITLE
Question: where is the previous setting `spellright.ignoreWords`

### ISSUE THREAD

**Issue by neilsustc** [bartosz-antosik/vscode-spellright#52]
I happen to have a look at my settings.json and find this

![image](https://user-images.githubusercontent.com/7588612/29495518-b75dfba0-85f3-11e7-896d-a44e4c1989d8.png)

**Comment by bartosz-antosik** [comment]
It's been replaced by `spellright.dict` files in User/Workspace settings. At version 1.0.36. These words should actually be transferred at first read of this particular configuration settings (if this is workspace settings file, but it does not look like one, from user settings you have to remove these manually) with a message telling the number of words and `Close` button ;-).

Also there are two new commands (F1) to open user/workspace dictionaries for easier edit/reference!

P.S. Is "IntelliSence" really spelled correctly? ;-)

**Comment by neilsustc** [comment]
Got it.

But I get this error from the console (when debugging this extension), why?

```
SpellRight found no bindings among these files:
d:/Workspace/vscode-spellright/lib/bin/spellchecker-darwin-1.6.6-x64.node
d:/Workspace/vscode-spellright/lib/bin/spellchecker-linux-1.6.6-x64.node
d:/Workspace/vscode-spellright/lib/bin/spellchecker-win32-1.6.6-x64.node
Activating extension `ban.spellright` failed: Cannot read property 'Spellchecker' of null.
Here is the error stack:  TypeError: Cannot read property 'Spellchecker' of null
    at Object.<anonymous> (d:\Workspace\vscode-spellright\lib\bindings.js:45:24)
```

P.S. I learn this word from vscode website 🤣

![image](https://user-images.githubusercontent.com/7588612/29499205-8196981a-863e-11e7-90be-ca3a189d44d0.png)

**Comment by bartosz-antosik** [comment]
Regarding problem loading bindings: Are you per chance using a developer version? Does it use Electron version 1.6.6? Other version? Could you please determine which?

**Comment by neilsustc** [comment]
Version 1.16.0-insider
Commit aebaece67321b35f133be393fe8adcd50333d5ee
Date 2017-08-18T05:22:55.860Z
Shell 1.7.4
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64

**Comment by bartosz-antosik** [comment]
Ok, so the Electron has been bumped to 1.7.4. I will prepare new binaries at most on the beginning of next week. That's the life of native module dependencies...

BTW: Are you on Windows? If so I should be able to prepare the binaries sooner.

**Comment by neilsustc** [comment]
Yes, on Windows.
There is no hurry. Take your time 👍

**Comment by bartosz-antosik** [comment]
Just published version 1.1.22 should have right bindings for insiders version of VSCode.

**Comment by neilsustc** [comment]
Thanks
